techniques/general/G176.html

```diff
@@ -2,196 +2,138 @@
+            <p>ある 1 秒間において 3 回<em>以上</em>の閃光を放つものがある (つまり、G19 の達成方法を用いることができない) が、閃光を放つ領域が視野 10 度 (これは視界における中心視野に相当する) の 25％未満である場合、自動的に達成基準を満たすことになる。
             </p>
-            <p>The 10 degree of visual field represents the central area of vision in the eye. This
-               area is highly packed with visual sensors. Flashes in this area are transmitted to
-               the visual cortex. For those with photosensitivity, this flashing of activity on the
-               visual cortex can cause seizures. Flashing on other areas of the eye (which have far
-               fewer sensors) has much less effect on the cortex. Hence, the focus on just the 10
-               degrees of central vision.
+            <p>視野 10 度は、視界における中心視野にあたる。この視野には、視覚センサーがたくさんある。この視野における閃光は、視覚野に送られる。光過敏性発作のある利用者は、視覚野上において閃光を受け取ることで発作を引き起こす恐れがある。その他の視野 (視覚センサーがずっと少ない部分) での閃光は、視覚野に与える影響がずっと少ない。そのため、中心視野の 10 度だけを対象とする。
             </p>
             <ul>
                
-               <li>If the content is for general Web use, you can use
-                  <a href="#smallsafe1">Formula 1: Small Safe Area for Web Content</a>.
+               <li>コンテンツを一般的なウェブで使用する場合には、<a href="#smallsafe1">公式 1: ウェブコンテンツにおける小さくて安全な領域</a>を用いることができる。
                </li>
                
-               <li>If the content is for a known display (e.g., in a company foyer) then
-                  <a href="#smallsafe2">Formula 2: Small Safe Area for Known Displays</a>
-                  should be used.
+               <li>コンテンツを既知のディスプレイ (例: 会社のロビーにあるディスプレイ) で使用する場合には、<a href="#smallsafe2">公式 2: 既知のディスプレイにおける小さくて安全な領域</a>を用いるべきである。
                </li>
                
             </ul>
             <p id="smallsafe1">
-               <strong>Formula 1: Small Safe Area for Web Content</strong>
+               <strong>公式 1: ウェブコンテンツにおける小さくて安全な領域</strong>
                
             </p>
-            <p>Most Web authors do not know how to translate visual field to pixels, which is what
-               they generally can deal with. This technique provides that translation.
+            <p>ほとんどのウェブコンテンツ制作者は、視野をピクセルに変換する方法を知らない。が、それは一般に対処可能なものである。この達成方法では、その変換方法を提供する。
             </p>
-            <p>At this point in time, the most prevalent display is 1024 x 768 and about 15-17 inches
-               diagonally. When viewed at a typical viewing distance (22-26 inches) a 10 degree visual
-               field will capture an area approximately 341 x 256 pixels. This is not circular, but
-               neither is the central vision of most users, and the difference is so small (and at
-               the edge of the central vision where sensors are fewer) that it is not important.
+            <p>この文書を執筆している時点では、最も一般的なディスプレイの解像度は 1024 x 768 ピクセルで、画面は約 15～17 型である。一般的な画面との距離 (56～66 センチメートル) で閲覧している際、視野 10 度はおおよそ 341 x 256 ピクセルの領域を占めることになる。これは円形ではないが、ほとんどの利用者の中心視野も円形ではない。その差異はとても小さいので (そして、中心視野でも視覚センサーの少ない端のことなので)、重要なことではない。
```

Comment:
別に型にする必要性もない、というか別箇所でインチが出てくるので

```suggestion
            <p>この文書を執筆している時点では、最も一般的なディスプレイの解像度は 1024 x 768 ピクセルで、画面は約 15～17 インチである。一般的な画面との距離 (56～66 センチメートル) で閲覧している際、視野 10 度はおおよそ 341 x 256 ピクセルの領域を占めることになる。これは円形ではないが、ほとんどの利用者の中心視野も円形ではない。その差異はとても小さいので (そして、中心視野でも視覚センサーの少ない端のことなので)、重要なことではない。
```